recipe/meta.yaml

```diff
@@ -1,38 +1,33 @@
 {% set name = "category_encoders" %}
-{% set version = "2.6.1" %}
+{% set version = "2.6.3" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: 51d4c594cd102a3d30a4a00f3ffa62f73b3d2f4298e7a57df8a8046a8d7881f5
-  patches:
-    - patches/replace_pkg_resources_with_importlib.patch
+  url: https://github.com/scikit-learn-contrib/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: efe91b537e7aa311fc3e6e7d61b553134fea4ba6b230da0d67d86ff5d5e7627e
 
 build:
   number: 0
   skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
-    - patch       # [not win]
-    - m2-patch    # [win]
   host:
     - python
     - pip
     - wheel
     - setuptools
   run:
     - python
-    - scikit-learn >=0.20.0
+    - scikit-learn >=1.0.0
```

Comment:
should stay as it was before >= 0.20.0 ?
https://github.com/scikit-learn-contrib/category_encoders/blob/26ef26106fcbadb281c162b76258955f66f2c741/setup.py#L36

Reply:
oh, must have been a mistake.